### PR TITLE
Ability to replay a game

### PR DESCRIPTION
**Pull request by rzulian**
This implements the ability to replay a game move by move. I need help on the formatting, but it works just fine.
screen shot 2018-10-02 at 10 28 45

### CODE REVIEW

File: src/components/Game.vue [hunk 1]
```diff
@@ -170,22 +176,37 @@ export default class Game extends Vue {
   @Prop()
   auth: string;
 
+  replayPrevMove() {
+   this.replayMove --;
```
Comment (coyotte508) [inline src/components/Game.vue]: Should not go below 0

File: src/components/Game.vue [hunk 2]
```diff
@@ -170,22 +176,37 @@ export default class Game extends Vue {
...
+   this.replay();
+  }
+
+  replayNextMove() {
+   this.replayMove ++;
```
Comment (coyotte508) [inline src/components/Game.vue]: Should not go beyond last move

File: src/components/Game.vue (hunk 1 above)
Comment (rzulian) [reply src/components/Game.vue]: ok

File: src/components/Game.vue (hunk 2 above)
Comment (rzulian) [reply src/components/Game.vue]: ok

**Pull request merged**